schema/deploy/shipping/views.sql

```diff
@@ -2576,4 +2580,313 @@ grant select
+  , enroll_details.received_this_season_flu_vaccine
+  , enroll_details.received_this_season_flu_vaccine_date
+  , enroll_details.novax_reason
+  , enroll_details.vaccine_method
+  , enroll_details.vaccine_where
+  , enroll_details.vaccine_hx
+  , enroll_details.covid_vaccine
+  , enroll_details.covid_novax_reason
+  , enroll_details.sought_care_base
+  , enroll_details.hall_health
+  , enroll_details.hospital_where_base
+  , enroll_details.hospital_ed_base
+  , enroll_details.hospital_arrive_base
+  , enroll_details.hospital_leave_base
+  , enroll_details.prior_test_base
+  , enroll_details.prior_test_positive_base
+  , enroll_details.prior_test_positive_date_base
+  , enroll_details.prior_test_type_base
+  , enroll_details.prior_test_result_base
+  , enroll_details.contact_base
+  , enroll_details.wash_hands_base
+  , enroll_details.clean_surfaces_base
+  , enroll_details.hide_cough_base
+  , enroll_details.mask_base
+  , enroll_details.distance_base
+  , enroll_details.travel_countries_phs_base
+  , enroll_details.travel_states_phs_base
+  , enroll_details.countries_visited_base
+  , enroll_details.states_visited_base
+  , enroll_details.tier_1
+  , enroll_details.tier_2
+  , enroll_details.tier_3
+  , enroll_details.swab_and_send_calc
+  , enroll_details.kiosk_calc
+  , enroll_details.alerts_off
+
+  from encounters encounter
+  join enrollments enroll on encounter.individual_id = enroll.enrollment_individual_id
+  join warehouse.site site on encounter.site_id = site.site_id
+  join warehouse.individual individual on individual.individual_id = encounter.individual_id
+  left join warehouse.primary_encounter_location enc_loc on enc_loc.encounter_id = encounter.encounter_id
+  left join warehouse.location loc on enc_loc.location_id = loc.location_id
+  left join shipping.age_bin_fine_v2 age_fine on age_fine.range @> encounter.age
+  left join shipping.age_bin_coarse_v2 age_coarse on age_coarse.range @> encounter.age
+  left join shipping.age_bin_decade_v1 age_decade on age_decade.range @> encounter.age
+  left join shipping.uw_reopening_enrollment_fhir_encounter_details_v1 enroll_details on enroll_details.encounter_id = enroll.enrollment_encounter_id
+  left join warehouse.sample sample on sample.encounter_id = encounter.encounter_id
+  -- Filter out follow up encounters
+  where not encounter.details @> '{"reason": [{"system": "http://snomed.info/sct", "code": "390906007"}]}'
+)
+;
+
+comment on view shipping.uw_reopening_encounters_v1 is
+  'For the UW reopening project, a view that ties enrollment questionnaire data to encounter data';
+
+revoke all
+on shipping.uw_reopening_encounters_v1
+from "incidence-modeler";
+
+grant select
+  on shipping.uw_reopening_encounters_v1
+  to "incidence-modeler";
+
+create or replace view shipping.uw_reopening_ehs_reporting_v1 as
+(
+  select ids.barcode
+  , results.collect_ts as sample_collection_date_time
+  , results.status_code as test_result
+  , results.result_ts as test_result_date_time
```

Comment:
Non-blocker, but I'd have a preference for naming these sample_collection_date and test_result_date as the values  are dates and not datetimes.

Reply:
Changed.